examples/validation/validate_points_markers.py

```diff
@@ -38,6 +38,67 @@
 scene = gfx.Scene()
 scene.add(gfx.Background.from_color("#bbb", "#777"))
 
+pygfx_sdf = """
+    let m_sqrt_3 = 1.7320508075688772;
+
+    // The coords below assume OpenGL coords, but y is flipped, so we flip the coords.
+    // Its also shifted so that the center of the triangle is at the origin.
+    var coord_for_sdf = vec2<f32>(coord.x, -coord.y) / size + vec2<f32>(0.5);
```

Comment:
There is always a bit of SDF black magic.... i tried to remove the "minus" sign, but i just get black now.

I'll see what I can do about this, but it feels like an optimization nobody asked for so I'm not so worried.